WcaOnRails/spec/models/merge_people_spec.rb

```diff
@@ -25,14 +25,14 @@
     person = FactoryGirl.create(:person_with_multiple_sub_ids)
     merge_people.person1_wca_id = person.wca_id
     expect(merge_people).to be_invalid
-    expect(merge_people.errors.messages[:person1_wca_id]).to eq ["This person has multiple subIds"]
+    expect(merge_people.errors.messages[:person1_wca_id]).to include "This person has multiple subIds"
```

Comment:
Basically I did this to be in parallel in the below (see why it has been changed) and also to make this to test the actual point of this spec.